python/servo/try_parser.py

```diff
@@ -151,7 +169,12 @@ def parse(self, input: str):
                 self.fail_fast = True
                 continue  # skip over keyword
             if word == "full":
-                words.extend(["linux", "linux-wpt", "linux-perf", "macos", "windows", "android", "ohos", "lint"])
+                words.extend(["linux", "linux-wpt", "linux-perf", "linux-prod-perf"])
+                words.extend(["macos", "macos-perf", "macos-prod-perf"])
+                words.extend(["windows", "windows-perf", "windows-prod-perf"])
+                words.extend(["android", "android-perf"])
+                words.extend(["ohos", "ohos-perf", "ohos-prod-perf"])
+                words.extend(["lint"])
```

Comment:
I think this will cause prod builds for every platform to be run on MQ (even though we skip bencher reports there). I think we should pack all perf into perf and/or prod-perf labels that would get expended to all these. Then on pushes to main add perf + prod-perf in addition to full.